src/plasma/plasma_extension.cc

```diff
@@ -357,26 +355,30 @@ PyObject *PyPlasma_receive_notification(PyObject *self, PyObject *args) {
    * object was added, return a tuple of its fields: ObjectID, data_size,
    * metadata_size. If the object was deleted, data_size and metadata_size will
    * be set to -1. */
-  int nbytes =
-      read_bytes(plasma_sock, (uint8_t *) &object_info, sizeof(object_info));
+  int64_t size;
+  int nbytes = read_bytes(plasma_sock, (uint8_t *) &size, sizeof(size));
+  CHECK(nbytes == sizeof(size));
```

Comment:
All of these `nbytes` should not be `nbytes` bcause `read_bytes returns an error code, so this check should fail.